### PR TITLE
Data Provider Page

### PR DESCRIPTION
**Pull request by muhammad-hassan11**

### CODE REVIEW

File: frontend/src/components/data-provider/Hero.tsx [hunk 1]
```diff
@@ -15,7 +16,7 @@ export default function Hero() {
               datasets.
             </p>
             <Button className="mb-16 bg-[#006064] px-6 py-3.5">
```
Comment (willy1989cv) [inline frontend/src/components/data-provider/Hero.tsx:18]: have to set asChild=true to the <Button> component so it renders button properties on its first child, which is the <Link>

File: frontend/src/components/data-provider/AddDataSection.tsx [hunk 2]
```diff
@@ -12,7 +13,9 @@ export default () => {
           individuals and organisations to contribute their valuable data to
           drive insights and innovations in sustainable transportation.
         </p>
-        <Button className="bg-[#006064] px-6 max-w-fit py-3.5">Add data</Button>
+        <Button className="max-w-fit bg-[#006064] px-6 py-3.5">
```
Comment (willy1989cv) [inline frontend/src/components/data-provider/AddDataSection.tsx:16]: have to set asChild=true to the component so it renders button properties on its first child, which is the <Link>

File: docs/data-provider/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+![How To](how_to.png)
+
+#### Featured Asset
+
+Allows the configuration of the element that appears on this section, which can be an **image**, an **mp4 video**, or a **YouTube embedded video**.
+**How to configure**: In the `public` folder of the frontend project, the following properties should be set in the `tdc-config.json` file to define the type and file path to be displayed:
+
+```
+{
+  "hero": {
```
Comment (willy1989cv) [inline docs/data-provider/README.md:32]: if its supposed to use the same as Home page you should implement the video section as its in Home page. maybe create a component

**Review by willy1989cv — CHANGES_REQUESTED** [review]

**Review by Gutts-n — APPROVED** [review]
LGTM

**Review by Gutts-n — CHANGES_REQUESTED** [review]
.

**Pull request merged**